presto-orc/src/main/java/com/facebook/presto/orc/Filters.java

```diff
@@ -503,25 +575,59 @@ public boolean testBytes(byte[] buffer, int offset, int length)
             }
             return false;
         }
+
+        @Override
+        public boolean equals(Object o)
+        {
+            if (this == o) {
+                return true;
+            }
+
+            if (o == null || getClass() != o.getClass()) {
+                return false;
+            }
+
+            MultiRange that = (MultiRange) o;
+            return Arrays.equals(filters, that.filters) &&
+                    Arrays.equals(longLowerBounds, that.longLowerBounds) &&
+                    nullAllowed == that.nullAllowed;
+        }
+
+        @Override
+        public int hashCode()
+        {
+            return Objects.hash(filters, longLowerBounds, nullAllowed);
+        }
+
+        @Override
+        public String toString()
+        {
+            return toStringHelper(this)
+                    .add("filters", filters)
+                    .add("nullAllowed", nullAllowed)
+                    .toString();
+        }
     }
 
-    public static class InTest
+    public static class BingintValues
             extends Filter
     {
-        static final long emptyMarker = 0xdeadbeefbadefeedL;
-        static final long M = 0xc6a4a7935bd1e995L;
-        private long[] longs;
-        int size;
+        private static final long emptyMarker = 0xdeadbeefbadefeedL;
```

Comment:
Is there a chance that a value can equal the emptyMarker?

Reply:
@elonazoulay Yes, that's possible. There is extra logic to handle this case.

Reply:
nit: `EMPTY_MARKER`

Reply:
@tdcmeehan Updated